lib/removePipelineExtras.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+var Cesium = require('cesium');
+var defined = Cesium.defined;
+
+module.exports = removePipelineExtras;
+
+function removePipelineExtras(gltf) {

```

Comment:
It can be in a separate PR, but glTF objects can be nested more than two levels deep, e.g., render states, and glTF allows adding `extras` to any glTF object property (including the root property).  For example, see: https://github.com/KhronosGroup/glTF/tree/master/specification#properties

The right way to handle this is to recursively walk into objects (not numbers/strings) and check for `extras`.  Base the code off of https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Source/Core/clone.js

Also, write unit tests.

Again, a separate PR is fine so we can merge this one.